src/api/crypto/blockchain/Imp.cpp

```diff
@@ -1077,18 +1068,14 @@ auto Blockchain::Imp::p2wpkh(
 {
     try {
         const auto& hrp = hrp_map_.at(chain);
-        const auto prog = [&] {
-            auto out = UnallocatedVector<std::uint8_t>{};
-            std::transform(
-                hash.begin(),
-                hash.end(),
-                std::back_inserter(out),
-                [](const auto& byte) {
-                    return std::to_integer<std::uint8_t>(byte);
-                });
-
-            return out;
-        }();
+        auto prog = UnallocatedVector<std::uint8_t>{};
+        std::transform(
```

Comment:
can we think about moving such transform into some tool class ? and call Hash.toUnallocatedVector i can imagine we can have multiple times such transform in the codebase

Reply:
I do not want to focus on anything other than just removing lambdas during this task